test/framed/std/math_test.clj

```diff
@@ -1,46 +1,62 @@
 (ns framed.std.math-test
   (:require [clojure.test :refer :all]
-            [framed.std.math :as m]
             [clojure.test.check :as tc]
             [clojure.test.check.generators :as gen]
             [clojure.test.check.properties :as prop]
-            [clojure.test.check.clojure-test :refer [defspec]]))
+            [clojure.test.check.clojure-test :refer [defspec]]
+            [framed.std.math :as std.math]))
 
 (deftest test-nonzero?
-  (is (m/nonzero? -1))
-  (is (m/nonzero? 1))
-  (is (not (m/nonzero? 0)))
-  (is (not (m/nonzero? nil))))
+  (is (std.math/nonzero? -1))
+  (is (std.math/nonzero? 1))
+  (is (not (std.math/nonzero? 0)))
+  (is (not (std.math/nonzero? nil))))
 
 (deftest test-divide-some
-  (is (nil? (m/divide-some 1 0)))
-  (is (= 1/2 (m/divide-some 1 2)))
-  (is (thrown? AssertionError (m/divide-some nil 2)))
-  (is (thrown? AssertionError (m/divide-some 10 nil))))
+  (is (nil? (std.math/divide-some 1 0)))
+  (is (= 1/2 (std.math/divide-some 1 2)))
+  (is (thrown? AssertionError (std.math/divide-some nil 2)))
+  (is (thrown? AssertionError (std.math/divide-some 10 nil))))
 
 (defspec test-divide-some-generative
   (prop/for-all [numer gen/pos-int
                  denom gen/s-pos-int]
     (is (= (/ numer denom)
-           (m/divide-some numer denom)))))
+           (std.math/divide-some numer denom)))))
 
 (deftest test-round-bigdecimal
-  (is (= 0.33M (m/round-bigdecimal 2 0.3333333333M)))
-  (is (= 0.3M (m/round-bigdecimal 1 0.3333333333M))))
+  (is (= 0.33M (std.math/round-bigdecimal 2 0.3333333333M)))
+  (is (= 0.3M (std.math/round-bigdecimal 1 0.3333333333M))))
 
 (deftest test-round-places
-  (is (= 3.14 (m/round-places 2 3.14159))))
+  (is (= 3.14 (std.math/round-places 2 3.14159))))
 
 (deftest test-mean
-  (is (= 4.5 (double (m/mean (range 10)))))
-  (is (nil? (m/mean []))))
+  (is (= 4.5 (double (std.math/mean (range 10)))))
+  (is (nil? (std.math/mean []))))
 
 (deftest test-median
-  (is (= 4.5 (double (m/median (range 10)))))
-  (is (= 5 (m/median (range 11))))
-  (is (nil? (m/median []))))
+  (is (= 4.5 (double (std.math/median (range 10)))))
+  (is (= 5 (std.math/median (range 11))))
+  (is (nil? (std.math/median []))))
 
 (deftest test-mode
-  (is (= 2 (m/mode [1 2 3 2 3 2])))
-  (is (= [2 3] (m/mode [1 2 2 2 3 3 3])))
-  (is (nil? (m/mode [1 2 3 4 5]))))
+  (is (= 2 (std.math/mode [1 2 3 2 3 2])))
+  (is (= [2 3] (std.math/mode [1 2 2 2 3 3 3])))
+  (is (nil? (std.math/mode [1 2 3 4 5]))))
+
+(deftest test-variance
+  (let [vs [2 4 4 4 5 5 7 9]]
```

Comment:
I feel like this entire namespace is missing tests for empty lists, nil, mixed doubles and floats etc.  We don't need to fix everything up front but at least for these new functions I think we should start testing all the real conceivable inputs, e.g. empty lists, sequences as well as vectors, etc.